db/mongo/client.go

```diff
@@ -0,0 +1,83 @@
+package mongo
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	uuid "github.com/satori/go.uuid"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/ahab94/ApnaSchool/db"
+	"github.com/ahab94/ApnaSchool/models"
+)
+
+var ctx, _ = context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Don't create a global variable for `ctx` instead use separately in methods below.